deploy/README.md

```diff
@@ -42,11 +42,30 @@ You need two DNS records that resolve to your machine's IP address:
 - Wikibase, e.g., "wikibase.example"
 - QueryService, e.g., "query.wikibase.example"
 
-## Setup
+## Installation
+
+### Option 1: Web-based Setup (Recommended for New Users)
+
+If you're setting up a new instance on a fresh VPS, you can use our web-based installer to simplify the process. This method installs all required dependencies, configures your environment, and launches the full Wikibase Suite via a browser-based setup wizard.
+
+1. **Log in to your VPS as root.**
+2. **Run the following one-liner:**
+
+   ```sh
+   bash -c "$(curl -fsSL https://raw.githubusercontent.com/wmde/wikibase-release-pipeline/refs/heads/deploy-setup-script/deploy/setup/setup.sh)"
```

Comment:
This is intended to call the web installer, right? I think we need to add the `--web` arg, atm it runs the cli... which is also nice btw :)

Reply:
Yes, I am inviting this question now that there is a CLI installer. I am not at all sure that the CLI installer shouldn't be the default, in fact I think for now the CLI installer probably should be. The case for the Web installer would be when initiated through Cloud Config / Init where there is no CLI level user interaction and the user waits until the Web setup becomes available on port 8888. This case is not something I've currently been talking about as I haven't in my own testing found it to be necessarily as simply to explain / guide a potentially inexperienced devops user through the steps to doing. So far it has felt more VPS platform agnostic to forward the "login as root on a fresh VPS instance and run this curl+script script" option as the easiest to not get wrong option.